src/components/AuthorViewpoint/FactListCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Fact } from "@/types/conversations.types";
+import FactCard from "@/components/AuthorViewpoint/FactCard";
+import { MagnifyingGlassIcon, PlusIcon } from "@heroicons/react/24/outline";
+
+type FactListCardProps = {
+    facts: Fact[];
+};
+
+export default function FactListCard({ facts }: FactListCardProps) {
+    return (
+        <div className="h-[calc(100vh-56px-69px-32px)] rounded-lg bg-neutral-100 px-7 py-4">
+            <h1 className="mb-1 text-lg font-semibold text-neutral-700">
+                事實
+            </h1>
+            <div className="mb-2 flex w-full items-center py-1 pr-[52px]">
```

Comment:
Please use Mantine's [Select](https://mantine.dev/core/select/) component instead of manually writing an input. Furthermore, for future reference, I would recommend using Mantine [Input components](https://mantine.dev/core/input/) for any sort of input.

Reply:
Why use Select here? Isn't that a search box?